PWGGA/GammaConv/AliAnalysisTRDEfficiency.h

```diff
@@ -2,90 +2,50 @@
-        //virtual void          Tracks(AliAODEvent* fAOD);
+        virtual Bool_t          checkPi0(TClonesArray* lst, AliAODConversionMother* pi0, Double_t tmp[16], Int_t lbl);
+        virtual Int_t           GetEventCuts(AliESDTrdTrack* trdtrack, TString clss);
+        virtual Bool_t          GetTrackCuts(AliESDtrack* track);
+        virtual Double_t        GetSagitta(AliESDTrdTrack* trdtrack);
+        virtual Double_t        GetRating(AliESDv0 *v0, AliESDtrack *track, AliESDTrdTrack *trdtrack);
         virtual void            Terminate(Option_t* option);
-        
-        
+
     private:
-        AliAODEvent*            fAOD;           //! input event
+        AliESDEvent*            fESD;           //! input event
         TList*                  fOutputList;    //! output list
         TH1F*                   fHistPt;        //! dummy histogram
-        
-        TFile*                  file;
-        TList*                  fConversionGammas;
-        
-        TH1F*                   fhm1pt2;
-        TH1F*                   fhNpttr;
-        TH1F*                   fhNptun;
-        TH1F*                   fhfA;
-        //include filter bits
-        
-        // v0
-        // all tracks
-        TH1F*                   fhR;
-        TH2F*                   fhRpt;
-        TH1F*                   fhMv;
-        TH1F*                   fhvpost;
-        // hqu tracks
-        TH1F*                   fhRhqu;
-        TH2F*                   fhRpthqu;
-        TH1F*                   fhMhqu;
-        TH1F*                   fhvposthqu;
 
-        //tracks
-        TH1F*                   fhtxv;           // track->Xv()
-        TH1F*                   fhtyv;
-        TH1F*                   fhtzv;
-        
-        // gamma
-        TH1F*                   fhgpt;
-        TH1F*                   fhgpttrd;
-        TH2F*                   fhgRpt;
-        TH2F*                   fhgRpttrd;      // actually hqu but too late to change
-        
-        TH2F*                   fhgMinvM;       // comparing the GetPhotonMass and the GetInvMass functions
-        TH1F*                   fhgR;
-        TH2F*                   fhgptM;
-        TH2F*                   fhgptMhqu;
-        
-        TH2F*                   fhgptQ;     // pt vs photon Quality
-        TH2F*                   fhgptQhqu;
-        TH2F*                   fhgetaphi;
-        TH2F*                   fhgetaphihqu;
-        
-        TH2F*                   fhgxy;
-        TH2F*                   fhgxyhqu;
-        
-        // v0 duagther particles
-        TH1F*                   fhdn;
-        TH1F*                   fhdpt;
-        
-        // n dimensional
-        THnSparse*              fhna;
-        THnSparse*              fhnp;
-        THnSparse*              fhnhqu;
-        
-        // all of these events have a reconstructed photon
-        THnSparse*              fhgevent2;      // events with 
+        AliV0ReaderV1*          v0reader;
```

Comment:
think about using //! for most member variables, for example, it should be applied for all histograms